salt/utils/error.py

```diff
@@ -5,11 +5,15 @@
 '''
 from __future__ import absolute_import
 
-# Import python libs
-try:
+import sys
+# True if we are running on Python 3.
+PY3 = sys.version_info[0] == 3
+
+if PY3:
+    import builtins
+    exceptions = builtins
```

Comment:
This can be `from salt.ext.six.moves import builtins as exceptions`

Reply:
@dwoz
Are you suggesting to just combine lines 13+14?
Then why not just `import builtins as exceptions`?
And should that also be applied to `_compat.py`?

Reply:
Using `salt.ext.six` also removes the need for the python 3 check. So 8-14 can become one line. Yes, the same can be done for lines 37-45 in `_compat.py`

Reply:
The code for `utils/error.py` is updated using this suggestion and re-tested in the same way as before.
I realize that changing `_compat.py` should not be done as part of this bugfix, as there is no bug there. So I did not update that one.